cloudant-http/src/main/java/com/cloudant/http/internal/interceptors/CookieInterceptor.java

```diff
@@ -192,7 +130,7 @@ public HttpConnectionInterceptorContext interceptResponse(HttpConnectionIntercep
                     }
 
                     if (renewCookie) {
-                        logger.finest("Cookie was invalid attempt to get new cookie.");
+                        logger.finest("Cookie was invalid. Will attempt to get new cookie.");
```

Comment:
As discussed, we should consider checking if the cookie is expired prior to making the request.